kernel/packages/gif-processor/processor.ts

```diff
@@ -69,14 +108,7 @@ export class GIFProcessor {
     this.GLctx.bindTexture(this.GLctx.TEXTURE_2D, DCL.GL.textures[texId])
 
     if (this.isWebGL1) {
-      this.GLctx.texImage2D(
-        this.GLctx.TEXTURE_2D,
-        0,
-        this.GLctx.RGBA,
-        this.GLctx.RGBA,
-        this.GLctx.UNSIGNED_BYTE,
-        image
-      )
+      this.GLctx.texImage2D(this.GLctx.TEXTURE_2D, 0, this.GLctx.RGBA, this.GLctx.RGBA, this.GLctx.UNSIGNED_BYTE, image)
     } else {
       this.GLctx.texImage2D(
```

Comment:
In WebGL 2, `texSubImage2D` has better performance. Not sure why we didn't do it like this in the first place.

Reply:
cause we where in a hurry to implement it cause we had like 3 days before an event or something like that :P